src/app/main/main.component.html

```diff
@@ -1,4 +1,10 @@
 <mat-toolbar color="primary" class="header">
+  <button mat-icon-button (click)="sidenav.toggle()" class="toggle-button" color="primary">
+    <mat-icon aria-label="Toggle Sidenav">
+      <img src="https://www.gstatic.com/images/icons/material/system/1x/vertical_split_white_24dp.png" srcset="https://www.gstatic.com/images/icons/material/system/1x/vertical_split_white_24dp.png 1x,
+                     https://www.gstatic.com/images/icons/material/system/1x/vertical_split_white_24dp.png 2x" alt="toggle sidenav">
```

Comment:
Typo: The second line should include "2x" in the URL, otherwise the image appears at half size on high-resolution displays.